Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeHost.h

```diff
@@ -111,6 +114,7 @@ class RemoteLayerTreeHost {
     HashMap<WebCore::PlatformLayerIdentifier, std::unique_ptr<RemoteLayerTreeNode>> m_nodes;
     HashMap<WebCore::LayerHostingContextIdentifier, WebCore::PlatformLayerIdentifier> m_hostingLayers;
     HashMap<WebCore::LayerHostingContextIdentifier, WebCore::PlatformLayerIdentifier> m_hostedLayers;
+    HashMap<WebCore::ProcessIdentifier, HashSet<WebCore::PlatformLayerIdentifier>> m_hostedLayersInProcess;
```

Comment:
Not sure what "in process" means in the name. It's a shame we have to have both `m_hostedLayers` and `m_hostedLayersInProcess`.

Reply:
m_hostedLayersFromProcess?